src/generators/new/infra/database/repositories/repositories.js

```diff
@@ -29,18 +29,26 @@ async function generateRepositories (generate, filesystem, db) {
   return requires
 }
 
-async function updateRepositories (generate, filesystem, db) {
-  const mongoPath = '/src/infra/data/repositories/baseRepository.js'
-  if (fs.existsSync(`${filesystem.cwd()}${mongoPath}`)) db = 'mongo'
+async function updateRepositories(generate, filesystem) {
+  const paths = {
+    mongo: '/src/infra/data/repositories/baseRepository.js',
```

Comment:
why do not have a file or folder exclusive for mongodb?

Reply:
Sincerely, I do not know. @italojs would you know why?

Reply:
when we had implemented the mongo feature, didnt exist the herbs2mongo
we have a issue to fix it https://github.com/herbsjs/herbs-cli/issues/47